test/system/200-pod.bats

```diff
@@ -362,6 +362,7 @@ EOF
     run_podman 125 pod create --infra-name "$infra_name"
     assert "$output" =~ "^Error: .*: the container name \"$infra_name\" is already in use by .* You have to remove that container to be able to reuse that name: that name is already in use" \
            "Trying to create two pods with same infra-name"
+    assert "$output" =~ "or use --replace to instruct Podman to do so" "Get a propmt to use --replace in case if name conflict"
```

Comment:
I suggest you add this to the assert above as this is one error message and it makes most sense to check it as one.